Formula/gitversion.rb

```diff
@@ -0,0 +1,32 @@
+class Gitversion < Formula
+  desc "Easy semantic versioning for projects using Git"
+  homepage "https://github.com/GitTools/GitVersion"
+  url "https://www.nuget.org/api/v2/package/GitVersion.CommandLine/3.6.2"
+  version "3.6.2"
+  sha256 "db05ebacfb76ddca26180f0c70060fbb89800e434d3d55b524785d0da37dbac8"
+
+  bottle :unneeded
+
+  depends_on "mono"
```

Comment:
Can you make this `=> :recommended`? We tend to do so to informally not bump into people who would rather use the upstream `mono` package, etc.